tests/garage/torch/policies/test_deterministic_policy.py

```diff
@@ -6,88 +6,84 @@
 from torch import nn
 
 from garage.tf.envs import TfEnv
-from garage.torch.modules import MLPModule
 from garage.torch.policies import DeterministicPolicy
 from tests.fixtures.envs.dummy import DummyBoxEnv
 
 
-class TestContinuousNNPolicies:
+class TestDeterministicPolicies:
     # yapf: disable
-    @pytest.mark.parametrize('obs_dim, act_dim, hidden_sizes', [
-        (1, 1, (1, )),
-        (2, 2, (2, )),
-        (3, 3, (3, )),
-        (4, 4, (1, 1)),
-        (5, 5, (2, 2)),
-    ])
+    @pytest.mark.parametrize('hidden_sizes', [
+        (1, ), (2, ), (3, ), (1, 1), (2, 2)])
     # yapf: enable
-    def test_get_action(self, obs_dim, act_dim, hidden_sizes):
+    def test_get_action(self, hidden_sizes):
         env_spec = TfEnv(DummyBoxEnv())
+        obs_dim = env_spec.observation_space.flat_dim
+        act_dim = env_spec.action_space.flat_dim
         obs = torch.ones([1, obs_dim], dtype=torch.float32)
-        nn_module = MLPModule(
-            input_dim=obs_dim,
-            output_dim=act_dim,
+
+        policy = DeterministicPolicy(
+            env_spec=env_spec,
             hidden_nonlinearity=None,
             hidden_sizes=hidden_sizes,
             hidden_w_init=nn.init.ones_,
             output_w_init=nn.init.ones_)
 
-        policy = DeterministicPolicy(env_spec, nn_module)
         expected_output = np.full([1, act_dim],
                                   fill_value=obs_dim * np.prod(hidden_sizes),
                                   dtype=np.float32)
-        assert np.array_equal(policy.get_action(obs), expected_output)
+        assert np.array_equal(policy.get_action(obs)[0], expected_output)
```

Comment:
If this is `get_action` instead of `get_actions`, shouldn't the dimension of the output be `(action_dim,)` already?

Reply:
Yes it is. The get_action method returns the action and an empty dictionary, and that's why the [0] is there

Reply:
Oh I see. It makes sense.